includes/class-migrator-cli-orders.php

```diff
@@ -622,4 +622,47 @@ private function set_line_item_taxes( $line_item, $shopify_line_item ) {
 
 		$line_item->set_taxes( $taxes );
 	}
+
+	/**
+	 * Saves the old payment method data into the subscription meta table
+	 *  so it can be used during the mapping update by
+	 *  Migrator_CLI_Payment_Methods::update_orders_and_subscriptions_payment_methods.
+	 *
+	 * @param WC_Order $order the order to be updated.
+	 * @param array $shopify_order the shopify order data.
+	 */
+	private function process_payment_data( $order, $shopify_order ) {
+		$response     = Migrator_CLI_Utils::rest_request( 'orders/' . $shopify_order->id . '/transactions.json' );
+		$transactions = json_decode( wp_remote_retrieve_body( $response ) );
+		$transaction  = $this->get_capture_transaction( $transactions->transactions );
+
+		if ( ! $transaction ) {
+			WP_CLI::line( WP_CLI::colorize( ' %RCapture transaction not found%n' ) );
+			return;
+		}
+
+		switch ( $transaction->gateway ) {
```

Comment:
Is there a reason we are using `switch .. case` instead of `if ... else here`? I see only `shopify_payments` and `default`. Are we expecting more additions in the future?

Reply:
Yeah we're expecting more payment gateways. There is another PR open for PayPal Express Checkout now #10